src/Screens/Flows/App/TabsManagerScreen/Components/TabViewCard.tsx

```diff
@@ -69,9 +69,10 @@ const baseStyles = (theme: ColorThemeType) => {
             height: 194,
             minWidth: 148,
             borderRadius: 12,
-            borderWidth: 3,
+            borderStyle: "solid",
+            borderWidth: 1,
```

Comment:
Make sure that `selected` has the borderWidth set to 3

Reply:
Addressed.